content/components/data-table.mdx

```diff
@@ -0,0 +1,598 @@
+
+May have a min and max width constraint
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+  <img
+    width="456"
+    role="presentation"
+    src="https://user-images.githubusercontent.com/2313998/205382519-b97666be-05bd-4449-a4d9-b924c3bbe70a.png"
+  />
+
+  <img
+    width="456"
+    role="presentation"
+    src="https://user-images.githubusercontent.com/2313998/205361220-078f56f6-0681-4e64-bab3-7860f179d066.png"
+  />
+</Box>
+
+#### Shrink
+
+Stretch to fill available space in the parent **or shrink** to fit in the available space in the parent
+
+May have a min and max width constraint
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+  <img
+    width="456"
+    role="presentation"
+    src="https://user-images.githubusercontent.com/2313998/205361223-f9becdb3-64ca-44b2-a350-2d33e2b20b1f.png"
+  />
+  <img
+    width="456"
+    role="presentation"
+    src="https://user-images.githubusercontent.com/2313998/205361225-fcd06af1-8087-429a-99fe-1cf3c98fa479.png"
+  />
+</Box>
+
+#### Auto
+
+The column is the width of its widest cell. Not intended for use with columns whose content length varies a lot because a layout shift will occur when the content changes.
+
+May have a min and max width constraint
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+  <img
+    width="456"
+    role="presentation"
+    src="https://user-images.githubusercontent.com/2313998/205361223-f9becdb3-64ca-44b2-a350-2d33e2b20b1f.png"
+  />
+  <img
+    width="456"
+    role="presentation"
+    src="https://user-images.githubusercontent.com/2313998/205361224-f39ba3d5-feb4-464a-9059-2784bb91eec9.png"
+  />
+</Box>
+
+#### Explicit width
+
+Will be exactly that width and will not grow or shrink to fill the parent
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205385940-7dca1d5d-b2ae-4faf-a656-5b2bbec7b85f.png"
+/>
+
+#### Action column
+
+The action column would not accept a width. Its min-width would be the width of the actions.
+
+#### Filling available space
+
+If all columns have width constraints set and the row doesn't fill the table container, the difference is added to the width of the last column.
+
+### Cells
+
+<Box display="flex" flexDirection="column" sx={{gap: 3, 'h4': { margin: 0 }}}>
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box flex={1}>
+
+#### Density
+
+- Condensed
+- Normal
+- Spacious
+
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361229-8f29b673-93d5-414a-8e77-aa7b74e3c4b1.png"
+/>
+
+</Box>
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box flex={1}>
+
+#### Text alignment
```

Comment:
Similarly, should we provide examples of when and when not to adjust default alignment?